gsa/src/web/pages/audits/dialog.js

```diff
@@ -71,7 +70,7 @@ const AuditDialog = ({
   maxHosts = DEFAULT_MAX_HOSTS,
   name = _('Unnamed'),
   policies = [],
-  policyId,
+  policyId = UNSET_VALUE,
```

Comment:
I am not sure about this change. Shouldn't we ensure that a valid policy is used instead of allowing the unset value?